### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by coaxial** [benjaminoakes/maid#242] [Ideas]
So I played with Midjourney a little today and generated a few options for the icon. They're [BY-NC-4.0](https://creativecommons.org/licenses/by-nc/4.0/legalcode) licensed. Any favourite?

![1](https://user-images.githubusercontent.com/2927869/227571910-1d9d91b6-13e2-41e7-973a-a69dd790c878.png)
![2](https://user-images.githubusercontent.com/2927869/227571917-2e1c8065-8839-407f-b224-78d721db559d.png)
![3](https://user-images.githubusercontent.com/2927869/227571923-5ad461a8-0771-4bc1-b025-e1c97c574cb6.png)
![4](https://user-images.githubusercontent.com/2927869/227571925-15e207b6-edb8-465c-ab0b-68e8f7750d61.png)
![5](https://user-images.githubusercontent.com/2927869/227571931-267158e4-6c60-4704-bfed-a7f1ba1508b8.png)
![6](https://user-images.githubusercontent.com/2927869/227571934-01a8e6fa-a858-487c-a46f-76417e311799.png)
![7](https://user-images.githubusercontent.com/2927869/227571941-4aabaea3-bb81-4700-8e7c-60af9f6bcd5f.png)
![8](https://user-images.githubusercontent.com/2927869/227571945-c5f640a0-10f5-43db-8500-a452df988edb.png)

**Comment by benjaminoakes** [comment]
1 and 4?  Perhaps best to have them icon-sized to help decide what's easily recognizable.

It might be nice to add some files and folders on the ground.  :smile:

**Comment by coaxial** [comment]
![resized_1](https://user-images.githubusercontent.com/2927869/227579961-89549d3d-1123-497b-9125-72515e64f071.png) ![resized_2](https://user-images.githubusercontent.com/2927869/227579970-1644faee-da80-4e8e-b571-25362ab0a727.png) ![resized_3](https://user-images.githubusercontent.com/2927869/227579973-a55b1c96-bcec-4933-8018-73c2280c24ce.png) ![resized_4](https://user-images.githubusercontent.com/2927869/227579976-c42e8061-70f4-4972-af06-d1646f101fb0.png) ![resized_5](https://user-images.githubusercontent.com/2927869/227579980-f376b9e5-668c-4b56-9351-3b7d94000f88.png) ![resized_6](https://user-images.githubusercontent.com/2927869/227579984-9e9396cb-0ad4-4bef-aad1-04c0b1f9a07c.png) ![resized_7](https://user-images.githubusercontent.com/2927869/227579987-60f7f465-120f-4561-9f6e-f3367f7b4fca.png) ![resized_8](https://user-images.githubusercontent.com/2927869/227579988-cbff8487-4909-4f88-b882-3efe579bd403.png)

I'll try and see if MJ can add elements to existing images.

**Comment by benjaminoakes** [comment]
3 seems very recognizable, but it looks more like a tablet or phone.  Almost everyone would run Maid on a laptop.  Could we change that?

**Comment by coaxial** [comment]
I tried getting MJ to add elements to existing images but it doesn't work well at all. I'm not very proficient with Photoshop or GIMP and wouldn't be able to do it myself :/ I liked 2, 3, 4 personally but yeah 3 looks a little like a tablet. I was going for an Apple ][ looking anthropomorphized computer. Prompt was something like `an anthropomorphized 1980s computer with arms and legs, holding a broom in its hand and wiping the floor. smile, icon, naive`

**Reply by benjaminoakes** [reply]
I could go for any of them, but image generation AI always comes up with crazy-looking text, so perhaps we should avoid those or try to erase it?  I could try that in GIMP sometime, but perhaps not soon.
